### ISSUE TITLE
Error building EzXML.jl on Julia v0.6.3

### ISSUE THREAD

**Issue by felipenoris** [bicycle1885/EzXML.jl#65]
This started happening yesterday.

```julia
julia> Pkg.update()
INFO: Updating METADATA...
INFO: Updating BusinessDays master...
INFO: Updating XLSX master...
INFO: Updating REPLTetris master...
INFO: Updating InterestRates master...
INFO: Updating ZipFile master...
INFO: Computing changes...
INFO: Installing EzXML v0.6.3
INFO: Building EzXML
Info: Directory /home/felipenoris/.julia/v0.6/EzXML/deps/usr/lib does not exist!
Info: Downloading https://github.com/bicycle1885/XML2Builder/releases/download/v1.0.0/XML2Builder.x86_64-linux-gnu.tar.gz to /home/felipenoris/.julia/v0.6/EzXML/deps/usr/downloads/XML2Builder.x86_64-linux-gnu.tar.gz...
[08:58:40] ######################################################################## 100,0%
Info: No hash cache found
Info: Calculated hash d3125723e5c75ded1eb01e5ce06c7dfae149ee276aa2d3f82a64edb2e8df23b8 for file /home/felipenoris/.julia/v0.6/EzXML/deps/usr/downloads/XML2Builder.x86_64-linux-gnu.tar.gz
Info: Installing /home/felipenoris/.julia/v0.6/EzXML/deps/usr/downloads/XML2Builder.x86_64-linux-gnu.tar.gz into /home/felipenoris/.julia/v0.6/EzXML/deps/usr
Inconsistency detected by ld.so: dl-version.c: 224: _dl_check_map_versions: Assertion `needed != NULL' failed!
ERROR: Build process failed.
Stacktrace:
 [1] build!(::Array{String,1}, ::Set{Any}, ::String) at ./pkg/entry.jl:629
 [2] build!(::Array{String,1}, ::Dict{Any,Any}, ::Set{Any}) at ./pkg/entry.jl:637
 [3] build(::Array{String,1}) at ./pkg/entry.jl:652
 [4] resolve(::Dict{String,Base.Pkg.Types.VersionSet}, ::Dict{String,Dict{VersionNumber,Base.Pkg.Types.Available}}, ::Dict{String,Tuple{VersionNumber,Bool}}, ::Dict{String,Base.Pkg.Types.Fixed}, ::Dict{String,VersionNumber}, ::Set{String}) at ./pkg/entry.jl:572
 [5] update(::String, ::Set{String}) at ./pkg/entry.jl:461
 [6] (::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#update,Tuple{String,Set{String}}})() at ./pkg/dir.jl:36
 [7] cd(::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#update,Tuple{String,Set{String}}}, ::String) at ./file.jl:70
 [8] #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{Any,N} where N) at ./pkg/dir.jl:36
 [9] update() at ./pkg/pkg.jl:228
```

**Comment by felipenoris** [comment]
I'm running on Ubuntu.

```
julia> versioninfo()
Julia Version 0.6.3
Commit d55cadc (2018-05-28 20:20 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: Intel(R) Core(TM) i5-6600K CPU @ 3.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, skylake)
```

**Comment by DilumAluthge** [comment]
We’re experiencing the same error on Travis Linux builds.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by felipenoris** [comment]
It happens only on Julia v0.6.3 , released in the last few days.
Build works fine on Julia v0.6.2 .

**Comment by joshualeond** [comment]
Just thought I'd add that there doesn't appear to be any problems with this package with `0.6.3` on Windows:

```julia
julia> versioninfo()
Julia Version 0.6.3
Commit d55cadc350* (2018-05-28 20:20 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, broadwell)
```

Went through all of the examples without any issues.

**Comment by felipenoris** [comment]
On mac it also works fine.

**Comment by bicycle1885** [comment]
I can reproduce the problem on Ubuntu VM. I think this and #64 are related. I'll take a look soon.

**Comment by Alexander-Barth** [comment]
I would like to release a new package depending on EzXML. Currently the package does not install  property on Linux because of this library issue. It seems that the problem of this issue has already been found (https://github.com/bicycle1885/EzXML.jl/pull/67#issuecomment-397495961).
Is there anything that could be done in helping to fix this issue?

And thanks for EzXML.jl: the XML namespace and XPath support is really great!

**Comment by bicycle1885** [comment]
I'm sorry for leaving these problems. I'm very busy these days but I'll work on it again this weekend.

**Comment by BenJWard** [comment]
@bicycle1885 If you have some idea what the issue is, but no time to sit down and deal with it, would you share any of your thoughts, as it might enable someone else to make a PR with a fix, taking some of the pressure off you :)

**Comment by bicycle1885** [comment]
I think the problem is in binary files distributed with EzXML.jl (https://github.com/bicycle1885/XML2Builder). I tried to rebuild them a few weeks ago but failed on some platforms. I have no idea at the moment.

**Comment by bicycle1885** [comment]
Fixed by #69.
